src/js/factory/weekView.js

```diff
@@ -153,15 +154,43 @@ module.exports = function(baseController, layoutContainer, dragHandler, options)
         /**********
          * 종일일정
          **********/
-        alldayView = new Allday(options.week, vLayout.getPanelByName('AllDay').container);
+        alldayPanel = vLayout.getPanelByName('AllDay');
+        alldayView = new Allday(options.week, alldayPanel.container);
         alldayView.on('afterRender', function() {
+            var collapseBtn = domutil.find(config.classname('.allday-collapse-button'), alldayView.container);
+
             vLayout.getPanelByName('AllDay').setHeight(null, alldayView.contentHeight);
+
+            if (collapseBtn) {
```

Comment:
I think that this click event listener must move to click handler of it.
`MonthClick.prototype._onClick` takes charge of handling click event in monthly view.
As you see, `AlldayClick.prototype._onClick` can have this event listener.